### ISSUE TITLE
Blank screen on built-in displays when external is attached

### ISSUE THREAD

**Issue by neverett8fr** [pop-os/pop#2021]
**Distribution (run `cat /etc/os-release`):**
Pop!_OS 21.04
64 bit
GNOME - 3.38.5
Windowing System - X11
 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Maybe an issue with GNOME - not sure

**Issue/Bug Description:**
Running Pop!_OS on my laptop with an additional external monitor.  
On suspending / turning the system off, then starting again - only the external screen is active (while the built-in turns off)  
The issue is quite minor - If it happens, I'm able to shut the laptop and open it again / re-suspend then log-on to resolve it.  

**Steps to reproduce (if you know):**
Log in  
Attach external monitor (mine is HDMI)  
Suspend session (also happens with locking the system)  
Close laptop  
Open again  
Only external monitor in use, while built-in is blank

**Expected behavior:**
On opening a laptop (with external monitor), both should turn on (not just the external)

**Other Notes:**
Maybe an issue with GNOME, etc.  
Minor issue as can close the laptop, then open it again to resolve.

**Comment by jacobgkau** [comment]
What model of laptop are you using?

**Comment by neverett8fr** [comment]
It's not got a brand name - but I can give you the specs if you'd like?  
Recently switched over from Windows which didn't have this issue, so I'm guessing it's a software issue - but no idea really.

**Comment by jacobgkau** [comment]
> It's not got a brand name - but I can give you the specs if you'd like?

Yes, please provide the CPU, GPU, storage, RAM, and motherboard model if you don't have a manufacturer and model name for the complete system. (A purchase link might include most of this-- I assume you had to buy the laptop somewhere?)

> Recently switched over from Windows which didn't have this issue, so I'm guessing it's a software issue - but no idea really.

System76 laptops do not exhibit this behavior, and neither do many other systems, so while it could be a software issue, it is related to the hardware you're using. It could also have to do with the laptop's firmware.

---

System76 customers can reach out to [support](https://support.system76.com/) for technical assistance. For non-System76 hardware, you can seek community support on [Reddit](https://reddit.com/r/pop_os/) or [Mattermost](https://chat.pop-os.org).

**Comment by neverett8fr** [comment]
```
CPU: Intel Core i7 8-Core 10875H  
RAM: 2x 16GB Corsair 2666MHz SODIMM DDR4  
GPU: NVIDIA GeForce RTX 2070 (laptop version)  
SSD: 2x 500GB Samsung 970 EVO PLUS M.2  
Motherboard: TongFang - GM7MP7P  
```
  
All the drivers (including the NVIDIA Graphics Card) are up to date.  
It's also worth noting that when running Ubuntu, I don't have the issue.  
  
And it's not too much of a problem (as it's quite intermittent), I just thought it's worth submitting an issue in-case others have had a similar problem.  
  
**And thanks for getting back to me so quick!**

**Comment by jacobgkau** [comment]
Thank you for the additional info. It looks like that laptop has switchable graphics (between Intel and NVIDIA)? Which graphics mode are you running the system in (NVIDIA, Integrated, Hybrid)? Does the issue occur in both NVIDIA and Integrated modes, or only one of those?

**Comment by neverett8fr** [comment]
I am running using NVIDIA graphics, I haven't tried the others.  
I can try and then get back to you later - but might take a while as it's not an issue that happens all the time.

**Comment by SuperAviation001** [comment]
do 
sudo cp ~/.config/monitors.xml  /var/lib/gdm3/.config/
